cmd/kubeadm/app/phases/bootstraptoken/clusterinfo/clusterinfo.go

```diff
@@ -69,7 +70,7 @@ func CreateBootstrapConfigMapIfNotExists(client clientset.Interface, kubeconfig
 
 	// Create or update the ConfigMap in the kube-public namespace
 	klog.V(1).Infoln("[bootstrap-token] creating/updating ConfigMap in kube-public namespace")
-	return apiclient.CreateOrUpdateConfigMap(client, &v1.ConfigMap{
+	return apiclient.CreateOrUpdate(context.TODO(), client.CoreV1().ConfigMaps(metav1.NamespacePublic), &v1.ConfigMap{
```

Comment:
you can drop the first parameter and just call ctx := context.Background() inside these new functions.
if we need to pass context around later it can be refactored.